---
layout: default
title: "Home"
group: "navigation"
order: 1
---

<script>
	window.setInterval(goCarousel, 4000, 'right')

	function goCarousel(direction) {
		console.log('carousel');
		console.log(direction);
		var elem = document.querySelector('.main-image.current')
		if (direction == 'left') { var next = elem.previousElementSibling || document.querySelectorAll('.main-image')[document.querySelectorAll('.main-image').length-1]; }
		else { var next = elem.nextElementSibling || document.querySelectorAll('.main-image')[0]; }
		elem.className = elem.className.replace(/\bcurrent\b/,'');
		next.className = next.className + ' current';
		
	}

</script>

<div id='image-holder'>
	<div class='carousel-ctrl' id='left-ctrl' onClick='goCarousel("left")'><</div>
	<div class='carousel-ctrl' id='right-ctrl' onClick='goCarousel("right")'>></div>
	<div id='carousel-img'>
		<img class='main-image current' src='public/images/utah_campus.jpg'></img>
		<img class='main-image' src='public/images/iobio.png'></img>
		<img class='main-image' src='public/images/genetics-main1.png'></img>		
	</div>	
	
</div>

<div class='container'>
	<div class='mission-statement'>
		<div style="font-size:24px; color:rgb(80,80,80)">
		Gabor Marth’s computational genomics laboratory is at the 
		<a href="http://ucgd.genetics.utah.edu">USTAR Center for Genetic Discovery</a>, 
		in the <a href="http://www.genetics.utah.edu">Department of Human Genetics</a>, at the 
		<a href="http://www.utah.edu">University of Utah</a>. We develop software tools 
		for genomic data analysis using a combination of biological, statistical, and engineering approaches 
		to accurately detect inherited genetic variations and somatic tumor mutations; to understand 
		tumor evolution at the subclonal level; and web-based, highly visual, real-time interactive tools 
		for intuitive analysis of genomic big data. Working together with  computational biologist colleagues at our Center, 
		 clinical collaborators at the <a href="http://medicine.utah.edu/">University of Utah Medical School</a> 
		and the <a href="http://www.huntsmancancer.org/">Huntsman Cancer Institute</a>, 
		diagnostic clinicians at the <a href="https://www.aruplab.com">ARUP</a> reference laboratory, we are building 
		computational tools for medical discovery and personalized medicine.
		<p></p>
		Our recent projects include the development of the popular <a href="http://arxiv.org/abs/1207.3907">FreeBayes</a>
		genetic variant caller program, built as part of the <a href="http://www.1000genomes.org">1000 Genomes 
		Project</a>, driving fast <a href="http://www.nature.com/nmeth/journal/v12/n10/full/nmeth.3505.html">whole-genome sequence analysis</a>. 
		We are developing innovative, reference-free and <a href="https://github.com/dillonl/graphite">graph-based</a> 
		methods for detecting <i>de novo></i> variants in families 
		and somatic mutations in tumors.  
		We our adapting our <a href="http://www.genomebiology.com/content/15/8/443">SubcloneSeeker</a> tumor 
		subclone reconstruction tool to elucidate subclone evolution in patients as they progress through 
		multiple courses of chemotherapy, for insight into the genetic causes for chemoresistance, and to offer treating oncologists
		guidance for choosing the next course of treatment. We are excited about our <a href="http://iobio.io">IOBIO</a> 
		project in which we developed a number of real-time analysis web apps for “at a glance” 
		inspection of large genomic sequence alignment and variant files (<a href="http://bam.iobio.io">bam.iobio.io</a> 
		and <a href="http://vcf.iobio.io">vcf.iobio.io</a>); 
		and for powerful, interactive, gene-focused disease variant analysis and prioritization 
		(<a href="http://gene.iobio.io">gene.iobio.io</a>). These tools form the underpinnings of intuitive, real-time interactive
		workflows for discovery and diagnostics.


	</div>
</div>
