---
layout: page
title: "Research"
permalink: research.html
group: "navigation"
order: 2
---

<div class='page-title'>Research</div>

<!-- Description of variant discovery research -->
<div class='mission-statement'>
	<div style="font-size:24px; color:rgb(80,80,80)">
		Variant discovery. The Marth laboratory has long-standing interest in genetic variant identification, starting with the 
		first statistically rigorous polymorphism discovery program, <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=10581034&dopt=Abstract">
		PolyBayes</a>. The current “incarnation” of PolyBayes is the popular short-variant caller program FreeBayes [link: arXive]. We have 
		developed software for mobile element insertion polymorphism discovery, Tangram [link: Tangram pub]. We are currently 
		working on reference-free, k-mer based approaches for de novo variant and somatic mutation detection (the Rufus project); 
		and a Variant Graph approach for variant adjudication and variant call set integration (Graphite [link: github]). We have 
		applied these tools in large population and medical sequencing projects including The SNP Consortium (TSC) [link], the HapMap 
		Project [link], the 1000 Genomes Project [link], the Human Genome Variation Consortium [link], and the Simon Foundation 
		Autism Research Initiative [SFARI, link].
	</div>
</div>

<!-- Tumour subclone research -->
<div class='section'>
	Tumor subclone analysis. We have strong interest in algorithm development to reconstruct the evolution of tumors across a cancer patient’s progression. We are actively developing our SubcloneSeeker software [link: pub] for analyzing multiple longitudinal tumor biopsies and multisite metastases. We are working with many research groups at the University of Utah, the Huntsman Cancer Institute, and elsewhere to hone our tools, and to empower the dissection of valuable cancer datasets.	
</div>

<!-- IOBIO description -->
<div class='section'>
	Web-based, real-time analysis of genomic big data. We have recently developed IOBIO [link: iobio.io], a powerful, web-based genomic data analysis platform, and multiple tools (or web apps) for rapid genomic data inspection and gene-based variant investigation. IOBIO is a major focus in the lab, with many new analysis apps in development. Our existing web tools are now in routine use in hundreds of research groups worldwide, deployed in Galaxy and in the International Cancer Genome Consortium [link].
</div>

<!-- GKNO description -->
<div class='section'>
	A flexible, UNIX command line driven tool launcher and pipeline execution system. Our GKNO [link] system allows analysts to use, out of the box, many best-practices sequence analysis pipelines, or to modify and tune these pipelines for their needs.
</div>
