---
layout: page
title: "Research"
permalink: research.html
group: "navigation"
order: 2
---

<div class='page-title'>Research</div>

<!-- Description of variant discovery research -->
<div class='mission-statement'>
	<div style="font-size:24px; color:rgb(80,80,80)">
		Variant discovery. The Marth laboratory has long-standing interest in genetic variant identification, starting with the 
		first statistically rigorous polymorphism discovery program, <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=10581034&dopt=Abstract">
		PolyBayes</a>. The current “incarnation” of PolyBayes is the popular short-variant caller program <a href="http://arxiv.org/abs/1207.3907">
		FreeBayes</a>. We have developed software for mobile element insertion polymorphism discovery, <a href="http://www.biomedcentral.com/1471-2164/15/795">
		Tangram</a>. We are currently working on reference-free, k-mer based approaches for de novo variant and somatic mutation detection (the 
		Rufus project); and a Variant Graph approach for variant adjudication and variant call set integration (<a href="https://github.com/dillonl/graphite">
		Graphite</a>). We have applied these tools in large population and medical sequencing projects including The SNP Consortium (TSC)
		the <a href="http://hapmap.ncbi.nlm.nih.gov/">HapMap Project</a>, the <a href="http://www.1000genomes.org/">1000 Genomes 
		Project</a>, the Human Genome Variation Consortium, and the <a href="http://sfari.org/">Simon Foundation Autism Research 
		Initiative</a>.
	</div>
</div>

<!-- Tumour subclone research -->
<div class='mission-statement'>
	<div style="font-size:24px; color:rgb(80,80,80)">
		Tumor subclone analysis. We have strong interest in algorithm development to reconstruct the evolution of tumors across a 
		cancer patient’s progression. We are actively developing our <a href="http://www.ncbi.nlm.nih.gov/pubmed/25160522">SubcloneSeeker</a>
		software for analyzing multiple longitudinal tumor biopsies and multisite metastases. We are working with many research groups at 
		the University of Utah, the Huntsman Cancer Institute, and elsewhere to hone our tools, and to empower the dissection of valuable 
		cancer datasets.	
	</div>
</div>

<!-- IOBIO description -->
<div class='mission-statement'>
	<div style="font-size:24px; color:rgb(80,80,80)">
		Web-based, real-time analysis of genomic big data. We have recently developed <a href="http://iobio.io">IOBIO</a>,
		a powerful, web-based genomic data analysis platform, and multiple tools (or web apps) for rapid genomic data inspection 
		and gene-based variant investigation. IOBIO is a major focus in the lab, with many new analysis apps in development. 
		Our existing web tools are now in routine use in hundreds of research groups worldwide, deployed in Galaxy and in the 
		<a href="https://dcc.icgc.org/">International Cancer Genome Consortium</a>.
	</div>
</div>

<!-- GKNO description -->
<div class='mission-statement'>
	<div style="font-size:24px; color:rgb(80,80,80)">
		A flexible, UNIX command line driven tool launcher and pipeline execution system. Our <a href="http://gkno.me">GKNO</a>
		system allows analysts to use, out of the box, many best-practices sequence analysis pipelines, or to modify and tune 
		these pipelines for their needs.
	</div>
</div>
